stock_quant_partner_info/migrations/8.0.1.1.0/pre-migration.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```

Comment:
Because this is the first module that gets to OCA, so no migration is needed. If you internally need it, update all your installations and then the script is useless